README.md

```diff
@@ -182,6 +182,10 @@ All files and folders should be written in `kebab-case`, with the exception of V
 | ------------ | ----------- | ----------- | ---------------------- |
 | /photos/\_id | /image/\_id | 301         | Nuxt server middleware |
 
+## Just
+
+If [`just`](https://github.com/casey/just) is your preferred command runner, you can also use `just run {script}` to run any pnpm script and `just` on its own to list the available scripts (e.g. `just run docker:build`).
+
```

Comment:
Some mention here of the motivation might be nice, i.e. parity with the other Openverse repositories that use Just.